core/api.py

```diff
@@ -99,3 +99,75 @@ def get_current_year_term(self):
                 break
 
         return {"yearterm": yearterm, "is_two_weeks_from_end": is_two_weeks_from_end}
+
+    def get_worker_id_from_byu_id(self, byu_id):
+        url = secret_settings.API_WORKER_ID_IAM_URL + "?byu_id=" + byu_id
+        headers = {"Authorization": self.build_auth_header()}
+        worker_id_request = requests.get(url, headers=headers)
+        worker_id_json_response = worker_id_request.json()
+        response_data = worker_id_json_response["data"]
+        worker_id = None
+        if len(response_data) > 0:
+            worker_id = response_data[0]["worker_id"]
+
+        return worker_id
+
+    # byu_id is passed into this so we can record it later. byu_id does not return in the response.
```

Comment:
Maybe make this comment a docstring? I'm not sure I understand what the comment means anyway, so maybe a little more precision/explanation?